src/make-styles.tsx

```diff
@@ -100,11 +110,12 @@ export const createNestedStyleObject = (cssDeclaration: string): NestedStyles =>
       if (nOpen === 0) {
         const declaration = cssDeclaration.substring(start, match.index).trim();
         start = match.index + 1;
-        if (name!.substring(0, 6) === '@media') {
```

Comment:
why not just do `if(name!.substring(0, 6) === '@media')` and then `else if(name!.substring(0, 10) === '@container') ...` seems more readable to me. I'd always avoid regex if not really necessary they are extremely hard to read.

Reply:
and I think we need to preserve the info if this was `@media` or `@container` here